### ISSUE TITLE
Alloying Recipes show "Element" instead of the actual elements in JEI on servers

### ISSUE THREAD

**Issue by darkphan** [CannoliCatfish/project-rankine#135]
See screen shot: https://i.imgur.com/8qzDkkA.png

Rankine Version: rankine-1.16.4-1.1.3
Forge: 35.1.15

**Comment by CannoliCatfish** [comment]
Thank you for your report!
I believe this problem relates with the version of forge you are using, as the version of Rankine you are using was made for Forge 35.1.10.  It seems to still work on version 35.1.13 as well.

It seems that in this later version of forge, the method by which the recipe populates the ingredients for alloying does not work on a server. I currently believe this to be the most likely culprit, but I will also continue to look into this more.

**Comment by darkphan** [comment]
Rolled the server to 35.1.13 as well as my client and its still doing the same thing :(

**Comment by CannoliCatfish** [comment]
Good to know and apologies for this problem. This seems to be a weird issue that probably stems from the ingredient handlers for the recipes, though its strange as when we tested we got it to work on one server, but not on a different one. The ingredient handler tries to obtain tags for the elements, but it appears to not work on the client when connecting to these servers.

If I cannot find a direct solution to the problem, I will make a fix (for a 1.1.4 update) which should allow for the base ingredient to show up in JEI instead of the "element" error item. Additionally, we will be adding some additional details in the Rankine Journal for alloys in case JEI does not work/a player does not have JEI installed. 

The recipes will be migrated to json format in the next major update among other changes, which will likely allow for an opportunity to re-write the code that might be causing the error and a better method of handling the ingredients in JEI.
